migrations/sql/V1.29.1__alter_mine_party_appt_type_code.sql

```diff
@@ -0,0 +1,46 @@
+
+ALTER TABLE public.mine_party_appt_type_code add person boolean NOT NULL DEFAULT 'false' ;
+ALTER TABLE public.mine_party_appt_type_code add organization boolean NOT NULL DEFAULT 'false' ;
+ALTER TABLE public.mine_party_appt_type_code add grouping_level int4 NOT NULL DEFAULT '1' ;
+COMMIT;
+
+INSERT INTO mine_party_appt_type_code
+    (
+    mine_party_appt_type_code,
+    description,
+    display_order,
+    create_user,
+    update_user,
+    person,
+    organization,
+    grouping_level
```

Comment:
Should this script also update the create_timestamp and update_timestamp columns? or will the default values throw dates in the table?

Reply:
create and update timestamps are populated by the table defaults. which uses now().